### PR TITLE
Providing basic examples 

### PR DESCRIPTION
**Pull request by scouturier**
Providing basic examples based on real-life usage

### CODE REVIEW

**Review by cnuland — APPROVED** [review]
LGTM

**Review by sabre1041 — DISMISSED** [review]
LGTM

File: deploy/examples/basic_automated_cr.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  name: quayecosystem
+spec:
+  quay:
+    imagePullSecretName: redhat-pull-secret
+    deploymentStrategy: Recreate
+    skipSetup: false
+    keepConfigDeployment: true
+    routeHost: quay.app.example.com
+  redis:
+    imagePullSecretName: redhat-pull-secret
```
Comment (sabre1041) [inline deploy/examples/basic_automated_cr.yml]: Typically, Redis would originate from a different registry than Quay/Clair (Red Hat Container Catalog vs Quay). By default, the operator specified _registry.access.redhat.com_ as the registry hosting Redis and this registry does not require authentication

**Review by sabre1041 — CHANGES_REQUESTED** [review]

File: deploy/examples/basic_automated_cr.yml (hunk 1 above)
Comment (sabre1041) [reply deploy/examples/basic_automated_cr.yml]: @scouturier bump ^^

Comment (scouturier) [reply deploy/examples/basic_automated_cr.yml]: I have updated both files to use the red hat repo rather than quay.

**Review by sabre1041 — APPROVED** [review]
LGTM

**Pull request merged**